assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -13,6 +21,39 @@ export default {
 
     withPluginApi('0.8.12', (api) => {
 
+      api.onToolbarCreate(toolbar => {
+        // create thumbnail selector button
+        toolbar.addButton({
+          trimLeading: true,
+          id: "select-preview-button",
+          group: "insertions",
+          icon: "id-card",
+          title: "select_preview_button",
```

Comment:
This isn't a translation key. 
<img width="110" alt="Screenshot at May 10 11-05-16" src="https://user-images.githubusercontent.com/5931623/57495926-bd5be580-7313-11e9-9205-edc6e33916ab.png">

Reply:
Ooops, yes meant to fix that, sorry.